concepticondata/conceptlists/Chacon-2019-220.tsv

```diff
@@ -0,0 +1,244 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	PORTUGUESE
+Chacon-2019-220-1	1	grate	3114	GRATE (SOMETHING)	raspar
+Chacon-2019-220-2	2	hairy			piloso (peludo
+Chacon-2019-220-3	3	he	1211	HE	ele
+Chacon-2019-220-4	4	instrumental			instrumental
+Chacon-2019-220-5	5	manioc flour in water			chibe
+Chacon-2019-220-6	6	manioc squeezer			tipiti
+Chacon-2019-220-7	7	good morning			bom dia
+Chacon-2019-220-8	8	river chanel			paranã
+Chacon-2019-220-9	9	porrige	3651	MUSH (FOOD)	mingau
+Chacon-2019-220-10	10	sand	3460	BEACH	Praia
+Chacon-2019-220-11	11	above	1741	ABOVE	encima
+Chacon-2019-220-12	12	afternoon	1807	AFTERNOON	tarde
+Chacon-2019-220-13	13	all	98	ALL	todos
+Chacon-2019-220-14	14	anaconda	896	ANACONDA (WATER BOA)	cobra-grande
+Chacon-2019-220-15	15	and	1577	AND	e
+Chacon-2019-220-16	16	animal	619	ANIMAL	animal
+Chacon-2019-220-17	17	arm	1673	ARM	braço
+Chacon-2019-220-18	18	ash	646	ASH	cinza
+Chacon-2019-220-19	19	back	1291	BACK	costas
+Chacon-2019-220-20	20	bad	1292	BAD	mal
+Chacon-2019-220-21	21	bark	1204	BARK	casca
+Chacon-2019-220-22	22	alive	1422	BE ALIVE	vivo
+Chacon-2019-220-23	23	born	1090	BE BORN	Nascer
+Chacon-2019-220-24	24	behind	1686	BEHIND	atrás
+Chacon-2019-220-25	25	belly	1251	BELLY	barriga
+Chacon-2019-220-26	26	below	1485	BELOW OR UNDER	em baixo
+Chacon-2019-220-27	27	big	1202	BIG	grande
+Chacon-2019-220-28	28	bird	937	BIRD	pássaro
+Chacon-2019-220-29	29	bite	1403	BITE	morder
+Chacon-2019-220-30	30	black	163	BLACK	preto
+Chacon-2019-220-31	31	blood	946	BLOOD	sangue
+Chacon-2019-220-32	32	blow	176	BLOW (WITH MOUTH)	soprar
+Chacon-2019-220-33	33	dull	379	BLUNT	embotado
+Chacon-2019-220-34	34	bone	1394	BONE	osso
+Chacon-2019-220-35	35	boy, child	1366	BOY	menino
+Chacon-2019-220-36	36	breast	1402	BREAST	seio
+Chacon-2019-220-37	37	burn	2102	BURN	queimar
+Chacon-2019-220-38	38	canoe	1970	CANOE	Canoa
+Chacon-2019-220-39	39	chest	1592	CHEST	peito
+Chacon-2019-220-40	40	clean	704	CLEAN	limpo
+Chacon-2019-220-41	41	cloud	1489	CLOUD	nuvem
+Chacon-2019-220-42	42	cold	1287	COLD	frio
+Chacon-2019-220-43	43	come	1446	COME	vir
+Chacon-2019-220-44	44	cook	1100	COOK (SOMETHING)	Cozinhar
+Chacon-2019-220-45	45	corpse	767	CORPSE	Cadáver
+Chacon-2019-220-46	46	correct	1725	CORRECT (RIGHT)	correto
+Chacon-2019-220-47	47	count	1420	COUNT	contar
+Chacon-2019-220-48	48	cut	1432	CUT	cortar
+Chacon-2019-220-49	49	dark	706	DARK	escuro
+Chacon-2019-220-50	50	day	1260	DAY (24 HOURS)	dia
```

Comment:
Thanks @CarolinHu, I'll leave that (and all of the other remarks) for @tresoldi to decide, also based on the discussions in #807.

Reply:
It is rather hard in Portuguese/Spanish, because in my impression they don't only colexify, but people can have a hard time distinguishing the concepts unless examples are given. Which of course can be a problem because in many native languages "day" actually colexifies with "sun" (as in [all?] Tupian languages).

I would be tempted to say that DAY (NOT NIGHT) is probably closer to what informants mean, but DAY (24 HOURS) is a safer assumption unless we can confirm from the data. The most important thing to consider is whether we will miss information when comparing to other "South American" concept lists or not.